espnet2/train/collate_fn.py

```diff
@@ -51,6 +51,9 @@ def __init__(
         rand_crop: bool = True,
         crop_audio: bool = True,
         not_sequence: Collection[str] = (),
+        kernel_size: float = 25,
```

Comment:
Kernel_size is ambiguous with the term in convolution. How about using frame or window size, and frame shift instead of stride?